docs/developing.md

```diff
@@ -63,13 +63,13 @@ npm i vtex-tachyons --save
 
 :loudspeaker: **Disclaimer:** In the course of this document we assume that you know how to use the basics of git as well like do a commit, rebase and merge. Before open a PR, read the [designing section](./designing.md) and take some minutes thinking if your change is appropriate to our styleguide.
 
-As we know, a Pull Request let you tell to others about changes you have pushed to a branch in a GitHub Repo. So, to avoid confusion when we discuss with you your changes, it's convenient to talk how we create our branches and make our commits. We strongly recommend that you follow our standard. Your PR will be accepted if you have two positive signs in the reviews made by VTEX team members.
+As we know, a Pull Request lets you tell to others about changes you have pushed to a branch in a GitHub Repo. So, to avoid confusion when we discuss with you your changes, it's convenient to talk how we create our branches and make our commits. We strongly recommend that you follow our standard. Your PR will be accepted if you have two positive signs in the reviews made by VTEX team members.
 
 ##### Branch
 
-As in many project the `master` branch must reflect what is running in production. All the development branches must be synchronized with the master through `git rebase` command.
+As in many projects the `master` branch must reflect what is running in production. All the development branches must be synchronized with the master through `git rebase` command.
```

Comment:
Will us change to use the `main` instead of `master`?

Reply:
Good question. I think it's a valid thing to do. Can you bring this up in our Slack or in an issue, @ArthurSampaio?